CHANGELOG.md

```diff
@@ -4,6 +4,60 @@
 
 ### Grafana Mimir
 
+* [CHANGE] Alertmanager: the following metrics are not exported for a given `user` when the metric value is zero: #9359
+  * `cortex_alertmanager_alerts_received_total`
+  * `cortex_alertmanager_alerts_invalid_total`
+  * `cortex_alertmanager_partial_state_merges_total`
+  * `cortex_alertmanager_partial_state_merges_failed_total`
+  * `cortex_alertmanager_state_replication_total`
+  * `cortex_alertmanager_state_replication_failed_total`
+  * `cortex_alertmanager_alerts`
+  * `cortex_alertmanager_silences`
+* [FEATURE] Querier: add experimental streaming PromQL engine, enabled with `-querier.query-engine=mimir`. #8422 #8430 #8454 #8455 #8360 #8490 #8508 #8577 #8660 #8671 #8677 #8747 #8850 #8872 #8838 #8911 #8909 #8923 #8924 #8925 #8932 #8933 #8934 #8962 #8986 #8993 #8995 #9008 #9017 #9018 #9019 #9120 #9121 #9136 #9139 #9140 #9145 #9191 #9192 #9194 #9196 #9201 #9212 #9225 #9260 #9272 #9277 #9278 #9280 #9281 #9342 #9343 #9367 #9368 #9371 #9398 #9399 #9403 #9417 #9418 #9419 #9420
+* [FEATURE] Experimental Kafka-based ingest storage. #6888 #6894 #6929 #6940 #6951 #6974 #6982 #7029 #7030 #7091 #7142 #7147 #7148 #7153 #7160 #7193 #7349 #7376 #7388 #7391 #7393 #7394 #7402 #7404 #7423 #7424 #7437 #7486 #7503 #7508 #7540 #7621 #7682 #7685 #7694 #7695 #7696 #7697 #7701 #7733 #7734 #7741 #7752 #7838 #7851 #7871 #7877 #7880 #7882 #7887 #7891 #7925 #7955 #7967 #8031 #8063 #8077 #8088 #8135 #8176 #8184 #8194 #8216 #8217 #8222 #8233 #8503 #8542 #8579 #8657 #8686 #8688 #8703 #8706 #8708 #8738 #8750 #8778 #8808 #8809 #8841 #8842 #8845 #8853 #8886 #8988
```

Comment:
I'm moving both MQE and ingest-storage out of the 2.14. As discussed in the #9406 we are still working on those features.

Reply:
We don't do that. We keep PRs that were merged in the release section, even if we are not announcing the feature in release notes or have documentation for it.

Reply:
For example "Querier: add experimental streaming PromQL engine," is already in 2.13 as well.

Reply:
Ok, fair. I've put them back and split the mentioned MQE by PRs that are and aren't in the release.